src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,184 @@
+        user.setAge(21);
+        Assertions.assertThrows(ImpossibleRegisterUserException.class,
+                ()-> {
+            registrationService.register(user);
+                });
+    }
+
+    @Test
+    void loginHaveSizeEqualsAllowableSize_Ok() {
+        user.setId(4l);
+        user.setLogin("sanya2");
+        user.setPassword("7907001767");
+        user.setAge(23);
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
+    }
+
+    @Test
+    void loginIsBiggerThanAllowableSize_Ok() {
+        user.setId(8l);
+        user.setLogin("sanya1256");
+        user.setPassword("7907001767");
+        user.setAge(23);
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
+    }
+
+    @Test
+    void ageOfUserLessThanAllowableAge() {
+        user.setId(78l);
+        user.setAge(15);
+        user.setLogin("VenoM!!!");
+        user.setPassword("yufgugfyufguy");
+        Assertions.assertThrows(ImpossibleRegisterUserException.class,
+                ()-> {
+            registrationService.register(user);
+                });
+    }
+
+    @Test
+    void ageOfUserEqualsAllowableAge_Ok() {
+        user.setId(12l);
+        user.setAge(18);
+        user.setLogin("JaxTaller!");
+        user.setPassword("yufgugfyufguy1123");
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
+    }
+
+    @Test
+    void ageOfUserBiggerThanAllowableAge_Ok() {
+        user.setId(17l);
+        user.setAge(45);
+        user.setLogin("JaxTaller!isAlive");
+        user.setPassword("yufgu121123");
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
+    }
+
+    @Test
+    void passwordLengthOfUserLessThanAllowableLength_NotOk() {
+        user.setId(77l);
+        user.setAge(24);
+        user.setPassword("gyfyt");
+        Assertions.assertThrows(ImpossibleRegisterUserException.class,
+                ()->{
+            registrationService.register(user);
+                });
+    }
+
+    @Test
+    void passwordLengthOfUserEqualsAllowableLength_Ok() {
+        user.setId(123l);
+        user.setAge(49);
+        user.setLogin("DyingLight");
+        user.setPassword("121123");
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
```

Comment:
Consider using `Assertions.assertDoesNotThrow` to verify that no exception is thrown for valid password lengths.